### ISSUE TITLE
[BUG] Ollama long running generate error

### ISSUE THREAD

**Issue by sebiweise** [jorge-menjivar/unSAGED#63]
**Describe the bug**
When using Ollama server and a Vercel deployment I´m running into some timeouts

**Screenshots**
![image](https://github.com/jorge-menjivar/unSAGED/assets/22999200/9e0eab28-1fda-4334-9ede-f0a7be597548)

**Error**
Error: Your function was stopped as it did not return an initial response within 25s

**Additional context**
Ollama server running on Hetzner Cloud with a Nginx Proxy Manager in front to proxy the ollama server to https and enable Access Lists for HTTP Basic Auth

**Comment by jorge-menjivar** [comment]
Is this different from #55?

**Comment by sebiweise** [comment]
Yes and no, the problem in #55 was the fact that the /api/models call had a timeout. But the /api/chat call has the same problem when using Ollama. Everything longer then just a "Test" is taking to long to respond and result into the error I have described here.
The example "Why is the sky blue?" was the prompt that wasn't working here so nothing special.

**Comment by jorge-menjivar** [comment]
Have you been able to narrow down the cause of this?

**Comment by sebiweise** [comment]
No not now, I will try to create a more powerfull server with a gpu, maybe that will make ollama more reliable.

**Comment by daniel-farina** [comment]
I just saw this, I guess it's fixed here now: https://github.com/jorge-menjivar/unSAGED/pull/81

Please give it a try it should be working now.
